.drone/scripts/mentors.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+# Import GPG Key.
+echo $debian_packaging_key | gpg --import-private-key
```

Comment:
Can you quote this variable and use the form `${debian_packaging_key}`? That's the form we use across the rest of the repository, and it would help with consistency.